nemo_skills/training/data_preparation_utils/filters.py

```diff
@@ -250,20 +288,21 @@ def process_dataset_entry(self, data_entry: str) -> str:
 
 
 class CodeTextFilter(BaseParallelProcessor):
-    def __init__(self, filter_type, solution_key='generation', **kwargs):
+    def __init__(self, filter_type, prompt_config, solution_key='generation', **kwargs):
```

Comment:
Alternatively, just accept the token right away without config file

Reply:
Better to use the template in my opinion given we might be switching models.